src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategyType.java

```diff
@@ -0,0 +1,25 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public enum KvsRetryStrategyType {
+    
+    DISABLED(0), EXPONENTIAL_BACKOFF_WAIT(1), UNKNOWN(-1);
```

Comment:
https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/65e38dac9b30523d43a57bc009d679e627b58d9a/src/utils/include/com/amazonaws/kinesis/video/utils/Include.h#L1542

I don't see an unknown present.

Reply:
Both, the addition of UNKOWN and the use of "m"s is to be uniform with how we defined the [StreamStatus enum](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/blob/a1953f10f8b382b0e10a2e8464f7645738e72294/src/main/java/com/amazonaws/kinesisvideo/producer/StreamStatus.java) which also has no UNKOWN in PIC.

Reply:
Yeah, I think it is a good idea to have the unknown in case something goes wrong with the translation